src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -313,5 +323,117 @@ public static AnnotatedCodeLocationImportanceVersionOne CreateAnnotatedCodeLocat
                     return AnnotatedCodeLocationImportanceVersionOne.Important;
             }
         }
+
+        public interface IBlameHunk
+        {
+            string Name { get; set; }
+
+            string Email { get; set; }
+
+            string CommitSha { get; set; }
+
+            int LineCount { get; }
+
+            int FinalStartLineNumber { get; }
+
+            bool ContainsLine(int line);
+        }
+
+        private class BlameHunk : IBlameHunk
+        {
+            private readonly string _Name;
+            private readonly string _Email;
+            private readonly string _CommitSha;
+            private readonly int _LineCount;
+            private readonly int _FinalStartLineNumber;
+
+            public BlameHunk(string name, string email, string commitSha, int lineCount, int finalStartLineNumber)
+            {
+                _Name = name;
+                _Email = email;
+                _CommitSha = commitSha;
+                _LineCount = lineCount;
+                _FinalStartLineNumber = finalStartLineNumber;
+            }
+
+            public string Name { get => _Name; set => throw new System.NotImplementedException(); }
```

Comment:
>set => throw new System.NotImplementedException(); [](http://example.com/codeflow?start=47&length=50)

nit: remove the set from interface. with that, you would not need to create this exception

Reply:
Fixed.